components/Slider/src/MDCSlider.h

```diff
@@ -221,6 +221,20 @@ IB_DESIGNABLE
  */
 @property(nonatomic, assign) BOOL shouldDisplayDiscreteValueLabel;
 
+/**
+ The color of the value label's text.
+
+ Defaults and resets to white.
+ */
+@property(nonatomic, strong, null_resettable) UIColor *valueLabelTextColor;
+
+/**
+ The color of the value label's background.
+
+ Defaults and resets to blue.
+ */
+@property(nonatomic, strong, null_resettable) UIColor *valueLabelBackgroundColor;
```

Comment:
Unfortunately, this is `null_resettable` because it originally was bound to `trackOnColor`, which was set by `primaryColor`, which was  `null_resettable` but not documented as such.